frontend-main/src/flowerCluster.ts

```diff
@@ -0,0 +1,76 @@
+/*
+花の束をクラスターとした時、そのクラスター内部における花の座標や回転、大きさについて配列として値を返すクラスです。
+現在はほぼランダムな値のみを返しますが、QRコードによって得られた情報により色々と変化させるように改造する予定です。
+*/
+
+import * as THREE from "three";
+
+export class flowerCluster {
+  private flowerOrigin: THREE.Vector3;
+  private flowerNumber: number;
+  private flowerClusterMaxRadius: number;
+  private flowerSizeOfMin: number;
+  private flowerSizeOfMax: number;
+
+  constructor(
+    flowerOrigin: THREE.Vector3,
+    flowerNumber: number,
+    flowerClusterMaxRadius: number,
+    flowerSizeOfMin: number,
+    flowerSizeOfMax: number
+  ) {
+    this.flowerOrigin = flowerOrigin;
+    this.flowerNumber = flowerNumber;
+    this.flowerClusterMaxRadius = flowerClusterMaxRadius;
+    this.flowerSizeOfMin = flowerSizeOfMin;
+    this.flowerSizeOfMax = flowerSizeOfMax;
+  }
+
+  //花の座標を取得する(ランダム)
+  getCoordinates(): THREE.Vector3[] {
+    const coordinates: THREE.Vector3[] = [];
+    for (let i = 0; i < this.flowerNumber; i++) {
```

Comment:
JS/TSには拡張for文というものがあるので、それを使うと良さそうです！
今回の場合、以下のようにすると見やすくなりそう

```suggestion
    for (const i in this.flowerNumber) {
```

`in`じゃなくて`of`を使う方法もあり、`in`では引数(index)が`i`に入るのに対し`of`では値(value)が`i`に入るという違いがあるので、気になったら調べてみてください！

Reply:
いえいえ、ありがとうございます！！次使うときに参考にさせていただきます